modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -71,6 +71,8 @@ using namespace cv::dnn::ocl4dnn;
 using namespace cv::dnn::cuda4dnn;
 #endif
 
+#include "fast_convolution/fast_convolution.hpp"
```

Comment:
Need to take a look on:
- test failures in Linux Debug configuration
- test failures in Linux AVX2 configuration (`-DCPU_BASELINE=AVX2`)
- looks like unconditional doubling of weights storage requires more memory and several Win32 tests started to fail with OOM message. @vpisarev

Reply:
Thanks for code reviewing. The failure in Linux Debug and Linux AVX2 only occurs in the quantized model. Since parameters of int8 layers rely on the output of fp32 models, we can modify the threshold to solve it in a short time.

For Win32, I'm looking for a way around it.